### PR TITLE
[Design System] DCSO - Create documentation - Toggle

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design system] DCSO - Create documentation - Toggle
* add CHANGELOG.md
* updated README.md
* add Documentation of props and component
* add different sizes of labels support

Before:
Знімок екрана 2023-07-21 о 18 57 51
Знімок екрана 2023-07-21 о 18 58 12

After:
Знімок екрана 2023-07-21 о 18 52 47
Знімок екрана 2023-07-21 о 18 53 01

## Links
- jira ticket: [DES-302](https://codedotorg.atlassian.net/browse/DES-302)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by levadadenys** [comment]
Waiting for https://github.com/code-dot-org/code-dot-org/pull/52680 to be reviewed and merged.

**Comment by levadadenys** [comment]
Need to merge https://github.com/code-dot-org/code-dot-org/pull/52753 to add different bodySize support for Toggle

File: apps/src/componentLibrary/toggle/Toggle.tsx [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+  /** The value attribute specifies the value of an input element. */
   value?: string;
+  /** Toggle label*/
   label?: string;
+  /** Is Toggle disabled */
   disabled?: boolean;
+  /** Toggle switch placement */
   position?: 'left' | 'right';
+  /** Size of Radio Button */
   size?: 'xs' | 's' | 'm' | 'l';
```
Comment (sanchitmalhotra126) [inline apps/src/componentLibrary/toggle/Toggle.tsx]: As a follow-up: would be great to share this type if multiple components are using it.

**Review by sanchitmalhotra126 — APPROVED** [review]

File: apps/src/componentLibrary/toggle/Toggle.tsx (hunk 1 above)
Comment (levadadenys) [reply apps/src/componentLibrary/toggle/Toggle.tsx]: yep, that was the plan, thanks

**Pull request merged**